clean/tangipahoa_da.py

```diff
@@ -3,19 +3,25 @@
 sys.path.append("../")
 import pandas as pd
 from lib.path import data_file_path
-from lib.columns import clean_column_names
+from lib.columns import clean_column_names, set_values
 from lib.uid import gen_uid
 
 
 def split_name(df):
     names = df.name.str.lower().str.strip().str.extract(r"(\w+) (\w+)")
     df.loc[:, "first_name"] = names[0]
     df.loc[:, "last_name"] = names[1]
-    return df
+    return df.drop(columns="name")
 
 
 def assign_agency(df):
-    df.loc[:, "agency"] = ""
+    df.loc[
+        (df.last_name == "gemar") & (df.first_name == "rodney"), "agency"
+    ] = "Hammond PD"
+
+    df.loc[
+        (df.last_name == "hampton") & (df.first_name == "mark"), "agency"
+    ] = "Hammond PD"
```

Comment:
What is "Hammond PD" doing here?

Reply:
DA offices are entities completely separate from PD or SO's, therefore, multiple law enforcement agencies can end up in one brady list. 

For example, the Baton Rouge DA brady list includes officers from Baton Rouge PD, Baton Rouge SO, Louisiana State PD, LSU University PD and the Department of Public Saffety. 

The brady list for Tangipahoa DA includes officers from Hammond PD and Tangipahoa SO because Hammond PD is an agency located in the city of Hammond which is itself located in the Tangipahoa region of Louisiana.